features/steps/tools.py

```diff
@@ -22,3 +25,31 @@ def env_is_true(value):
         # Otherwise, default to False
         else:
             return False
+
+
+def create_subnets(num):
+    subnet_list = []
+    ip_addresses = [
+        '10.30.40.0',
+        '192.168.200.0',
+        '172.20.0.0',
+        '10.40.50.0',
+        '192.168.150.0',
+        '10.50.60.0',
+        '192.168.75.0',
+        '172.30.0.0',
+        '10.60.70.0',
+        '192.168.250.0'
```

Comment:
I would probably define the subnet with the prefix also, so 192.168.250.0 -> 192.168.250.0/24, that way we could create subnets with different masks. We would have some regex that would parse the network address and the mask from this string